packages/celotool/src/cmds/deploy/upgrade/blockscout.ts

```diff
@@ -97,7 +97,7 @@ export const handler = async (argv: BlockscoutUpgradeArgv) => {
   )
 
   if (!isCelotoolHelmDryRun()) {
-    await createGrafanaTagAnnotation(argv.celoEnv, imageTag, dbSuffix)
+    await createGrafanaTagAnnotation(argv.celoEnv, imageTag, dbSuffix, blockscoutDBUsername)
```

Comment:
I think `blockscoutDBusername` won't work, we need this https://github.com/celo-org/celo-monorepo/blob/master/packages/celotool/src/lib/blockscout.ts#L97